### DISCUSSION TITLE
Add submenu in MenuDataItem

### DISCUSSION THREAD

**Discussion by AcernisSupport** [ant-design-blazor/ant-design-blazor#3486] [Q&A]
I implemented a project with the template as a study (not knowing ANTDesign Blazor well)
Inside I find the menu implemented like this
     private readonly MenuDataItem[] _menuData =
     {
         new MenuDataItem
         {
             Path = "/",
             Name = "welcome",
             Key = "welcome",
             Icon = "smile"
         }
     };
I tried to insert a submenu but I can't figure out how to do it
Does anyone have an example to help me understand?
Thank you and good job to everybody

Perini Luca

**Comment by ElderJames** [comment]
Thanks for contacting us @AcernisSupport , you can add a MenuDataItem list to the MenuDataItem.Children property.
